F02A02 Adventsaufgabe - Polylingual Pingu/Würgeschlangen/Würgeschlangen.py

```diff
@@ -108,22 +108,22 @@ def incubate(self, current_day: int) -> Union[Egg, Snake]:
 
 
 # let Adam and Eve lay 3 eggs
-for i in range(3):
+for _ in range(3):
     terrarium["eggs"].append(adam.breed(eve))
     print("🪺 An egg was laid!")
 
 
 # now incubate the eggs until all the baby snakes hatched
 print("The eggs will now be incubated")
 day = 0
-while not len(terrarium["eggs"]) == 0:
+while terrarium["eggs"]:
```

Comment:
PEP8 (https://peps.python.org/pep-0008/) is clear about using the collection directly to check for non-emptiness